### ISSUE TITLE
starting error

### ISSUE THREAD

**Issue by xishi00** [purcell/emacs.d#633]
Hi，Purcell!
When my Emacs finished its first initializing using this config,  it pops up the following error:
`Debugger entered--Lisp error: (wrong-type-argument number-or-marker-p 1.539419203172449e+21)
  timer-next-integral-multiple-of-time((23489 44099 72449 20000) 0.1)
  run-at-time(t 0.1 guide-key/polling-function)
  guide-key/turn-on-timer()
  guide-key-mode()
  run-hooks(after-init-hook delayed-warnings-hook)
  command-line()
  normal-top-level()`

My Emacs is updated yesterday with version 27.0.50 in ubuntu 18.04. How i fix this error?

**Comment by purcell** [comment]
Hmm, I've never seen this error before. That's not a release version of Emacs, so it's entirely possible it has a bug.

**Comment by xishi00** [comment]
I found this error is caused by the command:
`(add-hook 'after-init-hook 'guide-key-mode)` in init-editing-utils.el, line 324. When I comment this line, everything is ok. But the guide-key won't work anymore, even I run the command `guide-key-mode` in emacs, which it pops up an error as followings:
`Debugger entered--Lisp error: (wrong-type-argument number-or-marker-p 1.5394819912832815e+21)
  timer-next-integral-multiple-of-time((23490 41351 183281 322000) 0.1)
  run-at-time(t 0.1 guide-key/polling-function)
.......
  counsel-M-x()
  funcall-interactively(counsel-M-x)
  call-interactively(counsel-M-x nil nil)
  command-execute(counsel-M-x)`

I ignored some lines in the error, which makes the message very long.
